ui/main/src/app/modules/cards/components/card/card.component.ts

```diff
@@ -55,7 +86,113 @@ export class CardComponent implements OnInit, OnDestroy {
                     this.currentPath = urlParts[1];
                 }
             });
-        this.computeDisplayedDate();
+
+        this.userService.currentUserWithPerimeters()
+            .pipe(takeUntil(this.ngUnsubscribe))
+            .subscribe(userWithPerimeters => {
+                if (userWithPerimeters) {
+                    this.userWithPerimeters = userWithPerimeters;
+                    this.entityUser = this.userWithPerimeters.userData.entities[0];
+                    this.computeCountdown();
+                }
+            },
+                error => console.error(`something went wrong while trying to have currentUser with perimeters `)
+            );
+
+    this.computeDisplayedDate();
+    }
+
+    public isValidatelttd(): boolean {
+
+        console.log(`${this.lightCard.processInstanceId} =>  ${this.lightCard.lttd} :  ${this.handleDate(this.lightCard.lttd)}`);
+        if (this.entityUser !== undefined) {
+            if (!this.isArchivePageType() && this.lightCard.lttd != null
+                && this.lightCard.entitiesAllowedToRespond.includes(this.entityUser)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    isTimeToStartCountDown(): boolean {
+        const delta = this.calculateDiff();
+        return delta > 0 && delta < this.leftTime * 60;
+    }
+
+    startCountdownConfig() {
+
+        this.leftTime = this.configService.getConfigValue('feed.card.leftTime', false);
+        let leftTimeSeconds = this.leftTime * 60;
+        const endDate = new Date(this.calculateDiff()).getSeconds();
+
+        if (leftTimeSeconds > endDate) {
+            leftTimeSeconds = this.leftTime * 60;
+        } else {
+            leftTimeSeconds = endDate;
+        }
```

Comment:
I do not understand what this code is doing ? 
Why not just make : leftTimeSeconds = new Date(this.calculateDiff()).getSeconds() ;

Reply:
OK

Reply:
yes in fact it is enough just to call the method without getSeconds (), because the method getSecondsBeforeLttd (), already returns the time second.

        const leftTimeSeconds = this.getSecondsBeforeLttd() ;